bench-exchange/src/bench.rs

```diff
@@ -568,16 +576,30 @@ fn trader<T>(
         trades.chunks(chunk_size).for_each(|chunk| {
             let trades_txs: Vec<_> = chunk
                 .par_iter()
-                .map(|(signer, trade, side, src)| {
-                    let s: &Keypair = &signer;
-                    let owner = &signer.pubkey();
+                .map(|(owner, trade, side, src)| {
+                    let o: &Keypair = &owner;
```

Comment:
No one-letter variable names please. Also, I don't see any point in these lines. Why not write `&[&owner, &trade]` below?

Reply:
@garious This design is being followed in multiple places so, added it during copy-pasta. Anyways, I have taken more idiomatic approach now.

Reply:
Thanks!